app.js

```diff
@@ -32,15 +32,43 @@ const createDeleteButton = (todo) => {
         handleDelete(todo);
     })
     return deleteButton;
+}
+
+
+const handleEdit = (inputElement, editButton) => {
+        if(editButton.innerText === 'Edit'){
+            editButton.innerText = 'Update';
+            inputElement.removeAttribute("readonly");
+        }else{
+            editButton.innerText = 'Edit';
+            inputElement.setAttribute("readonly", "readonly");
+        }
+    
+    
+}
```

Comment:
1. Don't use `innerText` to compare. Use data attributes to compare
2. Use Early return. Avoid using `..else` when possible

```suggestion
const handleEdit = (inputElement, editButton) => {
        if(editButton.getAttribute("data-state") === 'edit'){
            editButton.innerText = 'Update';
            inputElement.removeAttribute("readonly");
            inputElement.setAttribute("data-state", "update");
            
            return
        }
        
        editButton.innerText = 'Edit';
        inputElement.setAttribute("readonly", "readonly");
        inputElement.setAttribute("data-state", "edit");
}
```